docs/_optimist/faq/index.en.md

```diff
@@ -107,3 +107,29 @@ Change the mountpoint in `/etc/fstab` to use the UUID after fetching the infos w
 
 We do not support multiattached volumes on our instances as cluster-capable file systems are required for multi attach volumes in order to handle concurrent file system access.
 Attempts to use multi-attached volumes without cluster-capable filesystems carries a high risk of data corruption, therefore this feature is not enabled on the Optimist platform.
+
+## Why am I unable to create a snapshot of a running instance?
+
+In order to provide consistent snapshots, the Optimist platform utilises the property [os_require_quiesce=yes](https://opendev.org/openstack/nova/commit/926e58a179ef373646164bea40dc46b1ebef4748).
+This property allows `fsfreeze` to suspend and resume access on running instances and ensures that a stable image is maintained on disk.
```

Comment:
ensures that a consistent image is created from the disk.

Reply:
Changed